auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -273,15 +277,27 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
       supportedEntityFlags.includes(business.nameRequest?.legalType) // Feature flagged Nr types
   }
 
-  /** Returns the Name value for the affiliation */
+  /** Returns the name of the affiliation */
   private name (item: Business): string {
-    return this.isNumberedIncorporationApplication(item) ? 'Numbered Benefit Company' : item.name
+    if (this.isNumberedIncorporationApplication(item)) {
+      switch (item.corpType.legalType) {
```

Comment:
@vysakh-menon-aot I'm using a new property here to determine what kind of numbered IA this is.